dubbo-common/src/main/java/org/apache/dubbo/common/config/ModuleEnvironment.java

```diff
@@ -94,19 +92,13 @@ public List<Map<String, String>> getConfigurationMaps(AbstractConfig config, Str
     @Override
     public Configuration getDynamicGlobalConfiguration() {
         if (dynamicConfiguration == null) {
-            return applicationDelegate.getDynamicGlobalConfiguration();
+            CompositeConfiguration configuration = new CompositeConfiguration();
```

Comment:
Composite application dynamic configuration and module ordered properties configuration